plugins/vue-example/frontend/public/javascripts/countly.models.js

```diff
@@ -0,0 +1,240 @@
+/*global $, countlyCommon, _, countlyVue */
```

Comment:
Codacy found an issue: ['$' is already defined as a built-in global variable.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=6339769)